### ISSUE TITLE
Allow input without quotes, add quotes automatically

### ISSUE THREAD

**Issue by mariusa** [josdejong/svelte-jsoneditor#177]
At https://www.freeformatter.com/json-formatter.html#before-output
I can paste this

``
{
age: 22
}
```

and it will format to add quotes for attributes, eg `"age"`

Please allow the same for https://jsoneditoronline.org/

**Comment by josdejong** [comment]
Hi Marius, thanks for your input. https://jsoneditoronline.org/ already has auto repair capability:

After pasting in text mode:

![afbeelding](https://user-images.githubusercontent.com/568626/197966081-d50e613f-8a2d-4a67-9f0c-840dba3cb17a.png)

Pasting in tree mode:

![afbeelding](https://user-images.githubusercontent.com/568626/197966273-1657300e-87ca-4977-941b-79686ae1e4f6.png)

Is that what you mean?

**Comment by mariusa** [comment]
Hi Jos, thanks. I see, so one has to paste with Tree mode, not with Text mode, which isn't intuitive.
Would it be possible to auto-repair after pasting within Text mode too?

**Comment by josdejong** [comment]
> so one has to paste with Tree mode, not with Text mode, which isn't intuitive.

Not really, all you have to do in text mode is click the big red "Auto repair" button bottom right.

What I find important is that the editor not "automagically" makes changes in content, but that you as a user are in control. It is an interesting idea though to automatically repair contents in text mode too, like in tree mode, as long as there is this clear message at the bottom allowing you to revert the changes and do it yourself. Let me think about it.

**Comment by mariusa** [comment]
>  click the big red "Auto repair" button bottom right.

Ah, I really missed that! That works, thanks! Feel free to handle as you wish then, or close the issue

**Comment by mariusa** [comment]
Perfect! Thanks for being open to feedback :)

**Comment by drik98** [comment]
Hey @josdejong I would really appreciate this feature. I currently have the problem that the pasted json content is not recognized as unformatted. I guess because it is not "compact" enough. If it was auto formatted anyway that would resolve this issue.
